mirtop/libs/parse.py

```diff
@@ -17,7 +17,8 @@ def parse_cl(in_args):
                 "export": _add_subparser_export,
                 "validate": _add_subparser_validator,
                 "spikein": _add_subparser_spikein,
-                "update": _add_subparser_update
+                "update": _add_subparser_update,
+		"convert": _add_subparser_convert
```

Comment:
I think `convert` is similar to `export`, so two options:

* use `export` and add `sql` as an option.
* instead of `convert` name it `mirtop sql` and use an option to say you want to create the database. then we can use `mirtop sql` to do the queries as well.

I prefer the second option.

Thanks!